### PR TITLE
Remove lodash dep in odsp driver

### PR DESCRIPTION
**Pull request by jatgarg**

### CODE REVIEW

File: packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts [hunk 1]
```diff
@@ -85,8 +83,7 @@ export class OdspSummaryUploadManager {
     ): Promise<IWriteSummaryResponse> {
         const { snapshotTree, blobs } = await this.convertSummaryToSnapshotTree(
             parentHandle,
-            // Clone as we change the blob contents.
-            cloneDeep(tree),
```
Comment (jatgarg) [inline packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts:89]: We are not changing the contents anymore due to removal of blob deduping, so removed cloning.

File: packages/drivers/odsp-driver/src/odspDocumentService.ts [hunk 2]
```diff
@@ -130,7 +127,6 @@ export class OdspDocumentService implements IDocumentService {
 
         this.hostPolicy = hostPolicy;
         if (this.odspResolvedUrl.summarizer) {
-            this.hostPolicy = cloneDeep(this.hostPolicy);
```
Comment (jatgarg) [inline packages/drivers/odsp-driver/src/odspDocumentService.ts:133]: Didn't get the purpose of this. Please check if any.

Comment (vladsud) [reply packages/drivers/odsp-driver/src/odspDocumentService.ts:133]: We are modifying it on the next line, so clone here is to ensure we did not modify the object that caller has and might continue to use. So maybe clone shallow via

this.hostPolicy = {...this.hostPolicy};

**Review by vladsud — APPROVED** [review]

File: packages/drivers/odsp-driver/src/odspDocumentService.ts (hunk 2 above)
Comment (jatgarg) [reply packages/drivers/odsp-driver/src/odspDocumentService.ts:133]: hmm, makes sense.

**Comment by jatgarg** [comment]
Yeah, I think we will be able to reduce the size of driver by a lot with this change, lets see.

**Pull request merged**